src/servers/ZoneServer2016/zoneserver.ts

```diff
@@ -6768,12 +6771,6 @@ export class ZoneServer2016 extends EventEmitter {
       );
       return;
     }
-    const vehicle = this._vehicles[vehicleGuid];
-    if (vehicle._resources[ResourceIds.FUEL] + fuelValue > 10000) {
```

Comment:
ok split this in another pr if you want this do be merge since it allow players to waste fuel while being at 100%